sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Demo/Program.cs

```diff
@@ -33,5 +38,52 @@
     return $"Hello World! OpenTelemetry Trace: {Activity.Current?.Id}";
 });
 
+#if !NETFRAMEWORK
+app.MapGet("/StressTest", () =>
+{
+    RunStressTest();
+    return "StressTest!";
+});
+#endif
+
 app.Run();
 #endif
+
+#if !NETFRAMEWORK
+void RunStressTest()
+{
+    // Get the number of available processors
+    int numProcessors = Environment.ProcessorCount;
+
+    Task[] tasks = new Task[numProcessors];
+
+    // Start each task
+    for (int i = 0; i < numProcessors; i++)
+    {
+        tasks[i] = Task.Run(() => Compute(cancellationTokenSource.Token));
+    }
+
+    var timeStamp = DateTime.Now.AddSeconds(20);
+    while (DateTime.Now < timeStamp)
```

Comment:
this will keep cpu busy for 20 secs..that was the intention, right?
If yes, modify the comment to say so!

Reply:
This will be reverted before merging.